CONTRIBUTING.md

```diff
@@ -42,12 +42,11 @@ Contributing to tests:
 ### Debugging Spec Tests
 
 - To fix errors always focus on passing all minimal tests first without running mainnet tests.
-- Spec tests often compare full expected vs actual states in JSON format. To better understand the diff it's convenient to use mocha's option `--inline-diffs`.
-- A single logical error can cause many spec tests to fail. To focus on a single test at a time you can use mocha's option `--bail` to stop at the first failed test
-- To then run only that failed test you can run against a specific file as use mocha's option `--grep` to run only one case
+- A single logical error can cause many spec tests to fail. To focus on a single test at a time you can use vitest's option `--bail 1` to stop at the first failed test
+- To then run only that failed test you can run against a specific file as use vitest's option `-t` to run only one case
 
 ```sh
-LODESTAR_PRESET=minimal ../../node_modules/.bin/mocha --config .mocharc.spec.yml test/spec/phase0/sanity.test.ts --inline-diffs --bail --grep "attestation"
+../../node_modules/.bin/vitest --run test/unit --bail 1 -t produceBlindedBlock
```

Comment:
From my experience if you only want to run a single test it is best to point to that file as it avoids skipping a bunch of tests which takes a bit, see related [discord discussion](https://discord.com/channels/593655374469660673/1179716009284870194)

```suggestion
LODESTAR_PRESET=minimal yarn vitest --run test/spec/phase0/sanity.test.ts --bail 1 -t "attestation"
```

Used an example that is as close to previous command as possible but we can definitely also use a different one

Reply:
Does this stil exist `test/spec/phase0/sanity.test.ts` ?
Previous command looked outdated, even before the recent move to vitest.

Reply:
The file is located here `packages/beacon-node/test/spec/presets/sanity.test.ts` but it seems like a bad example anyways, have to download spec (`yarn download-spec-tests`) beforehand manually to make them work. 

But even then it does not seems to work for me, getting the following error
```
FAIL  test/spec/presets/sanity.test.ts [ test/spec/presets/sanity.test.ts ]
ReferenceError: describe is not defined
```

@nazarhussain don't we have to import `describe` now? It is used here without an import
https://github.com/ChainSafe/lodestar/blob/9262064a9b5878d2b2e390a9e775c2cec8b1b1aa/packages/beacon-node/test/spec/utils/specTestIterator.ts#L144

Reply:
Looks like the `beacon-node` package hasn't been fully migrated yet, so that probably is the culprint.

Reply:
should be migrated now, may be worth a revisit